src/Mvc/Mvc.Core/src/MvcOptions.cs

```diff
@@ -379,6 +380,19 @@ public int MaxModelBindingRecursionDepth
     /// <value>Defaults to <c>8192</c>.</value>
     public int MaxIAsyncEnumerableBufferLimit { get; set; } = 8192;
 
+    /// <summary>
+    /// Gets or sets a value that determines if <see cref="CultureInfo.CurrentCulture"/> should always
+    /// be used to parse form values.
```

Comment:
If this property is `true`, then `CurrentCulture` will always be used to parse form values, regardless of the `type` attribute value. But, if we made this property the inverse of what it is currently (where a value of `true` implies the new behavior), then I think it would make sense to apply this suggestion (and also change `CurrentCulture` to `InvariantCulture`). There is some discussion in the [original issue](https://github.com/dotnet/aspnetcore/issues/6566#issuecomment-1213428561) about changing the name to `CultureInvariantFormValueFormattingEnabled`. Maybe we should do that and then apply this suggestion?

Reply:
My nit was in error and I didn't remove it before submitting my Pending comments. Sorry.

I very much prefer `false` by default for `bool`s in our public API but am certainly willing to be ignored / overridden / corrected.